wofi-pass

```diff
@@ -217,7 +219,11 @@ function get_passname_from_menu() {
     local wofi_ret
     local wofi_args=()
 
```

Comment:
Please extends the test for get_passname_from_menu and check if `--insensitive` really passed to wofi. See tests/test_get_passname

Reply:
all tests successful on my end